### ISSUE TITLE
Mistakes in the docstrings of cam2pixel and pixel2cam 

### ISSUE THREAD

**Issue by thatbrguy** [kornia/kornia#1170]
Hey everyone. I was going through the codebase and I noticed a few mistakes in the docstrings of the functions `cam2pixel` and `pixel2cam` which are available in `kornia/geometry/camera/pinhole.py`.

The function `cam2pixel` mentions the following about the return value in its docstring:
https://github.com/kornia/kornia/blob/9d594687db704845c3668d76b6e481666277e375/kornia/geometry/camera/pinhole.py#L582-L584

But on reading the code of the function `cam2pixel`, it looks like the output is in pixel coordinates. The output can exceed the range `[-1, 1]`. The function `cam2pixel` is used in `kornia/geometry/depth.py`. Here, we can see that the range normalization is applied to `pixel_coords_src`, which is the output of `cam2pixel`.

https://github.com/kornia/kornia/blob/9d594687db704845c3668d76b6e481666277e375/kornia/geometry/depth.py#L304-L310

Also, the function `pixel2cam` mentions the following about the return value in its docstring:
https://github.com/kornia/kornia/blob/9d594687db704845c3668d76b6e481666277e375/kornia/geometry/camera/pinhole.py#L557-L559

But on reading the code of the function `pixel2cam`, it seems like the output is in XYZ camera coordinates. The docstring mentioned that the output would be a `tensor of (u, v, 1) cam coordinates` which does not make sense. The output seems to be (x, y, z) cam coordinates. 

In summary, I think the return value descriptions in the docstrings of `cam2pixel` and `pixel2cam` are incorrect. I have not tried running the code though, I'm just mentioning my observations based on my reading. Do let me know your thoughts! I would be happy to send in a PR to fix this if the issue is verified.

**Comment by edgarriba** [comment]
@thatbrguy thanks for pointing this out. You are right, seems an issue with the docs. Feel free to send a fix - besides I'm seeing that we don't have any test for those two functions. Would be nice if you could add  it to verify.

**Comment by thatbrguy** [comment]
Great! I'll send in a PR to correct the docstrings. 

I'm not sure what specific tests could be added to verify the output of those functions though since their outputs can be unbounded. Perhaps some generic tests could be added (like gradcheck or some shape check). Also perhaps a test where we could pass the output of `pixel2cam` to `cam2pixel` to see if the output of `cam2pixel` is consistent with the input to `pixel2cam`. I could maybe try writing these tests and include them in the PR which corrects the docstring. Do let me know your thoughts.

**Comment by edgarriba** [comment]
Absolutely, this simple test could work. Go ahead with the PR. Feel free to join our slack so that we can give you support.

**Comment by thatbrguy** [comment]
Sure, will do!

**Comment by thatbrguy** [comment]
Closing since issue was handled in #1211.
